src/opencc.cc

```diff
@@ -43,9 +43,10 @@ class Opencc {
   opencc::DictPtr dict_;
 };
 
-Opencc::Opencc(const string& config_path) {
+Opencc::Opencc(const path& config_path) {
   opencc::Config config;
-  converter_ = config.NewFromFile(config_path);
+  // OpenCC accepts UTF-8 encoded path.
+  converter_ = config.NewFromFile(config_path.u8string());
```

Comment:
This change fixes the wrong path encoding given to opencc  on Windows.
Opencc does it UTF-8 to ANSI string conversion internally.